guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -15,46 +15,65 @@ New hosts are provisioned with a boot disk or PXE-less discovery image that {Pro
 PXE-less Provisioning with Discovery::
 New hosts use an ISO boot disk that loads the {Project} Discovery service. This service identifies hardware information about the host and lists it as an available host to provision.
 
+ifeval::["{build}" == "foreman"]
+NOTE: Discovery workflows are only available when Discovery plugin is installed.
+endif::[]
+
 .BIOS and UEFI Support
 
-With {ProjectName}, you can perform both BIOS and UEFI based PXELinux provisioning.
+With {ProjectName}, you can perform both BIOS and UEFI based PXE provisioning.
```

Comment:
At many places we use PXELinux (which is one implementation of PXE) instead PXE.